Express Beta APIs.postman_collection.json

```diff
@@ -3656,71 +4096,61 @@
 								},
 								"method": "PUT",
 								"header": [
-									{
-										"warning": "This is a duplicate header and will be overridden by the Authorization header generated by Postman.",
-										"key": "Authorization",
-										"value": "Bearer {{accessToken}}",
-										"description": "Requires the `access_token` for the hosting environment."
-									},
 									{
 										"key": "Content-Type",
-										"name": "Content-Type",
-										"type": "text",
 										"value": "application/json"
+									},
+									{
+										"key": "Authorization",
+										"value": "Bearer {{accessToken}}"
 									}
 								],
 								"body": {
 									"mode": "raw",
-									"raw": "{\n    \"id\": \"ChEiD3JvYm90LW5hbWVzcGFjZRIZCg5FbWFpbFRlbXBsYXRlcxCAgICAr8iACg\",\n    \"body\": \"\\n# Welcome Modified \\n\\nIf you have any questions or issues with your account please contact us.\\n\\nThanks!\\n\\n{{organization.name}}\\n\\nIf you did not make this request, please contact us.\",\n    \"bodyRendered\": \"<style data-embed>@import url(\\\"https://fonts.googleapis.com/css?family=Roboto:300,400,500\\\");</style><div style=\\\"font-family: 'Roboto',sans; text-align: center;\\\"><h1 style=\\\"color: #455369; font-weight: 300;\\\">Welcome</h1>\\n<p>If you have any questions or issues with your account please contact us.</p>\\n<p>Thanks!</p>\\n<p>{{organization.name}}</p>\\n<p>If you did not make this request, please contact us.</p>\\n</div>\",\n    \"description\": \"Confirms the user has verified their email address or registered the app\",\n    \"enabled\": false,\n    \"from\": \"{{{organization.name}}}\",\n    \"fromAddress\": \"noreply@sample-org.com\",\n    \"name\": \"Welcome\",\n    \"styles\": \"#body{font-family:'Roboto',sans;text-align:center}a{color:#004b9b;text-decoration:none}a:hover{text-decoration:underline}h1{color:#455369;font-weight:300}.btn{background-color:#2d9ee0;border-radius:4px;color:#fff;display:inline-block;font-size:14px;font-weight:500;letter-spacing:1px;margin:20px;padding:16px;text-transform:uppercase;text-decoration:none}.btn:hover{background-color:#2192d4;text-decoration:none !important}\",\n    \"subject\": \"Welcome to {{{organization.name}}}\",\n    \"type\": \"welcome\",\n    \"urlLifetime\": 0\n}"
+									"raw": "{\n  \"host\":\"{{smtpHost}}\",\n  \"mandatoryTLS\": true,\n  \"port\": \"{{smtpPort}}\",\n  \"username\": \"{{smtpUsername}}\",\n  \"password\": \"{{smtpPassword}}\"\n}"
```

Comment:
In _Configure/Modify SMTP Server_, the smtpPort parameter needs to not be in quotes, or it will be interpreted as a string (which won't work).  It should be as follows:

```
{
  "host":"{{smtpHost}}",
  "mandatoryTLS": true,
  "port": {{smtpPort}},
  "username": "{{smtpUsername}}",
  "password": "{{smtpPassword}}"
}
```

Reply:
Yup, I see that now, and have deleted the quotes -- I presume `port` is defined as an integer. 

FWIW, it works both ways, with and without quotes (which begs the Q if there's a bug).